var/spack/repos/builtin/packages/giflib/package.py

```diff
@@ -24,6 +24,16 @@ class Giflib(MakefilePackage, SourceforgePackage):
 
     patch('bsd-head.patch')
 
+    @property
+    def build_targets(self):
+        targets = ['all']
+        if self.spec.satisfies('@5.2.0:'):
+            targets.append('PREFIX={0}'.format(self.spec.prefix))
+            targets.append('LIBMAJOR={0}'.format(self.spec.version.up_to(1)))
+            targets.append('LIBVER={0}'.format(self.spec.version))
```

Comment:
Can you group this together in a single method, since it's the same code as for the `install` target?

Reply:
@alalazo not sure if this is a good idea, but did it anyway.

Reply:
Which concern do you have?

Reply:
I think this decreases readability, but I also see reason behind your suggestion: adding common parameters in one place reduces error chance